chapter2/tests/t2/chapter2-t2-gen-kustomization.sh

```diff
@@ -0,0 +1,53 @@
+#
+# generate_repository -- takes a repository name and generates a repository
+#
+# usage: generate_repository <NAME_REPOSITORY>
+#
+generate_repository() { 
```

Comment:
I think this function should be better named like 

```suggestion
generate_repository_kustomization_config() { 
```

or smth similar, so it would be clear what it does from its name

Reply:
done